h2/src/main/org/h2/table/InformationSchemaTable.java

```diff
@@ -1975,9 +1975,15 @@ private void routines(SessionLocal session, ArrayList<Row> rows, String catalog)
                         } else {
                             routineType = "FUNCTION";
                         }
+                        String externalName = null;
+                        String javaClassName = alias.getJavaClassName();
+                        String javaMethodName = alias.getJavaMethodName();
+                        if (!StringUtils.isNullOrEmpty(javaClassName) && !StringUtils.isNullOrEmpty(javaMethodName)) {
```

Comment:
These properties can only be `null` if an alias with source code was specified. It means only simple null check should be performed here, I don't see any good reasons to use `isNullOrEmpty()`, it can only confuse casual readers.

Reply:
Thank you for your comment and quick answer!